source/_includes/common-tasks/enable_entities.md

```diff
@@ -0,0 +1,17 @@
+## Enabling entities
+
+Some entities are disabled by default. To enable them, follow these steps:
+
+1. Go to **{% my integrations title="Settings > Devices & Services" %}** and select the integration of interest.
+1. Select **entities**.
+1. From the list of entities, select the entity you want to enable.
+    ![Select entity from list](/images/screenshots/enable_entities_01.png)
+1. In the pop-up, select the cogwheel.
+1. If you want to use this entity with a voice assistant, you need to expose it to the assistant.
+   - Select **Voice assistants**.
+    ![Enable entities](/images/screenshots/enable_entities_00.png)
+   - Enable the toggle button for all assistants that you want to use with this entity.
+   ![Expose entities to assistants](/images/screenshots/enable_entities_02.png)
+1. Go back and toggle the **Enabled** button.
```

Comment:
Should this step go above the voice assistant one?

As in, the focus for this instruction is how to enable entities; exposing them to a voice assistant feel secondary?

wdyt?

Reply:
yes, agreed. It is illogical. I tried to do it this way at first, too. The problem is when you follow this sequence: 
- first enable the toggle button
- then go into voice assistants and expose the device
- then leave the voice assistant page
the enable button will be disabled again. You have to toggle it again to enable the entity.
If you don't pick up on that, you think you have the entity enabled, but it isn't.

(At least that's the behavior I'm seeing on my system on b5 right now)

That's why I chose this sequence of events.

Reply:
Right, but that makes me wonder, why is the voice assistant step here in the first place? The voice assistant will have its defaults already when enabling?

Reply:
yes, removing that step is an option. I was hesitant about putting it in.